components/benefits_pane.js

```diff
@@ -232,6 +232,12 @@ export class BenefitsPane extends Component {
             </Grid>
           </Grid>
 
+          {nonFilteredBenefits.length === 0 ? (
+            <div>{this.resultsHeader(1, t("B3.search_tip"))}</div>
+          ) : (
+            ""
```

Comment:
I think it's preferable to use `null` rather than `""`, because I think `""` can sometimes (or maybe always?) cause an empty element to be included in the DOM